driver-rocketmq/rocketmq.yaml

```diff
@@ -18,7 +18,7 @@
 #
 
 name: RocketMQ
-driverClass: io.openchaos.driver.rocketmq.RocketMQChaosDriver
+driverClass: io.openchaos.driver.rocketmq.RocketPubSubDriver
 stateClass: io.openchaos.driver.rocketmq.RocketMQChaosState
```

Comment:
RocketMQChaosState config could be removed use a strategy pattern when you start controller